### ISSUE TITLE
Latex disappears

### ISSUE THREAD

**Issue by nchagnet** [aledeg/FreshRSS-extensions#9]
Hi,

I just installed the extension, enabled it, went to check on one of my feeds featuring heavy inline math LaTeX (hep-th arxiv, I can give a specific example if necessary).

What happened then is that when I load the page, I can very briefly see the inline math _unformated_ and it disappears (presumably when MathJax is loaded).

By the way, I am using the yunohost version of FreshRSS.

**Comment by aledeg** [comment]
Could you provide the feed so I can see what is going on?
Thank you

**Comment by ghost** [comment]
Yes, the feed is accessible here: http://export.arxiv.org/rss/hep-th

These are pictures of before/after turning on the extension. https://postimg.cc/gallery/YQc90hw

**Comment by aledeg** [comment]
What browser are you using? It seems that I cannot reproduce on Firefox.
Here is a screenshot of my output.
![Screenshot_2021-01-24 (39) hep-th updates on arXiv org · FreshRSS](https://user-images.githubusercontent.com/3056148/105636239-7a1e0f00-5e35-11eb-87aa-752fb0ab9236.png)

**Comment by ghost** [comment]
I was using Firefox but I just checked on Chrome and the issue is there as well. I also ran multiple tests:

- This issue happens as well when I run the MathJax Bookmarklet.
- Inspecting the source, I can make the LaTeX appear by toggling off the accessibility based aria-hidden.
- Looking for this issue in the MathJax community, I read it could be related to the fast preview module.

I would like to specify that the LaTeX is present in the source, it just seems to disappear.
If I could access another install of FreshRSS I could see if I can reproduce this unfortunately the demo.freshrss.org website does not allow me to add customs feeds which have LaTeX.

**Comment by aledeg** [comment]
@Alkarex Do you have any thoughts?

**Comment by aledeg** [comment]
@nchagnet Do you have any other FreshRSS extension enabled?

**Comment by ghost** [comment]
I had the default ones but I tried disabling them (as well as all other browser extensions) and it didn't help.

**Comment by aledeg** [comment]
I am running out of ideas why it's working on my machines and not on yours.
What theme are you using?

**Comment by Alkarex** [comment]
I have made a quick test and it worked for me, thought with some CSP errors - maybe something to look into.
Will check again a bit later

**Comment by Alkarex** [comment]
Any error/warning in the Web browser console?

**Comment by Alkarex** [comment]
Could you make share a test account with us with that feed and relevant extension enabled?
If yes, my email can be [found online](https://alexandre.alapetite.fr/cv/).

**Comment by ghost** [comment]
It's done, thank you so much!

**Comment by Alkarex** [comment]
@aledeg There seems to be at least two distinct bugs, in CSS and in JS:

For CSS, try with those changes:

```css
mjx-assistive-mml {
	/* position: absolute !important; */
}

mjx-container[jax="CHTML"] {
	line-height: 0;
	display: inline-block;
}
```

Furthermore, there is a race condition, as the extension does not wait for Mathjax to be loaded (this ends up working sometimes, not always):
![image](https://user-images.githubusercontent.com/1008324/105750561-31557b80-5f45-11eb-9c88-09aa7c1de866.png)

**Comment by aledeg** [comment]
Thank you for your help. I'll look into it.

**Comment by Alkarex** [comment]
@nchagnet Meanwhile, you can try to add a custom stylesheet (for instance with the CustomCSS extension) with the following:

```css
#global mjx-assistive-mml {
	position: relative !important;
}
mjx-container[jax="CHTML"] {
	display: inline-block;
}
```

**Comment by ghost** [comment]
Thank you very much! It worked perfectly!

**Comment by aledeg** [comment]
I found the JS error. It will be fixed in next release.
I still don't understand the CSS error :/
